src/Database.php

```diff
@@ -201,7 +197,7 @@ public function __toString()
      * @throws InvalidArgumentException for parameter/option parsing errors
      * @throws DriverRuntimeException for other driver errors (e.g. connection errors)
      */
-    public function aggregate(array $pipeline, array $options = [])
+    public function aggregate(array $pipeline, array $options = []): Traversable
```

Comment:
I would expect a `CursorInterface` like `Collection::aggregate()`

Reply:
Good point, will update.

Reply:
For context, when `Database::aggregate()` was implemented in https://github.com/mongodb/mongo-php-library/commit/c2922d30f2122c285b4a75db3b8b114b6e233350, the [Aggregate operation](https://github.com/mongodb/mongo-php-library/blob/c2922d30f2122c285b4a75db3b8b114b6e233350/src/Operation/Aggregate.php) conditionally returned non-Cursor Iterator implementations crafted from inline responses. That is certainly no longer relevant.